packages/ui-components/src/Inputs/TokenSelection/TokenSelection.tsx

```diff
@@ -233,7 +103,44 @@ const TokenSelection: React.FC<TokenSelectionProps> = ({
         />
       </SearchWrapper>
       <TokensList>
-        <TokenListValues />
+        {displayCurrencies.length ? (
+          displayCurrencies.map((currency, idx) => {
+            const { icon: Icon, id, name, symbol } = currency.view;
+            const selected = selectedToken?.view.id === id;
+            const onClick = selected
+              ? undefined
+              : () => {
+                  onClose?.();
+                  onChange(currency);
+                };
+
+            return (
+              <ListItem key={id} idx={idx} selected={selected} onClick={onClick}>
+                <ListItemAvatar>
+                  <Tooltip title={<h2>{`Add ${symbol} to MetaMask`}</h2>}>
+                    <Avatar
+                      className='list-item-avatar'
+                      children={Icon}
+                      onClick={() => onAddToMetaMask(id)}
+                      style={{ cursor: 'copy' }}
+                    />
+                  </Tooltip>
+                </ListItemAvatar>
+                <ListItemText className='list-item-text'>
+                  <Typography variant={'h6'} component={'span'} display={'block'}>
+                    <b>{symbol}</b>
+                  </Typography>
+                  <Typography variant={'body2'} color={'textSecondary'} display={'block'}>
+                    <b>{name}</b>
+                  </Typography>
+                </ListItemText>
+                {selected && <CircleStatus />}
+              </ListItem>
+            );
+          })
+        ) : (
+          <ListEmpty text='Not found any available tokens' />
```

Comment:
Instead of: 'Not found any available tokens'

Use: `No tokens available' or 'No available tokens found for selected chain. Please select a different chain to proceed.'